### ISSUE TITLE
missing graphic elements?

### ISSUE THREAD

**Issue by babslgam** [karl-kraus/legalkraus-archiv#3]
did some graphic elements get lost somewhere?
example file:
https://github.com/karl-kraus/legalkraus-archiv/blob/main/data/editions/D_000019-012-001.xml#L115-L120

**Comment by csae8092** [comment]
is missing in the "old" application/data as well; https://www.kraus.wienbibliothek.at/legalkraus/lesefassung/208689

**Comment by babslgam** [comment]
yes, but the old app doesn't use openseadragon with paging, so a simple fallback for missing image was enough.
will there be any more curation on the tei files?
